sig-and-wg/sig-core/proposals/event-bus-monitoring-metrics.md

```diff
@@ -0,0 +1,113 @@
+# Event Bus Monitoring Metrics Proposal
+
+## Event Bus
+- **Throughput**
+  
+  Number of all events delivered that were successfully received by publish-app per minute
+
+- **event propagation time 99 percentile**
+
+  event propagation time 99 percentile
+
+- **lag**
+
+  number of events in queue
+
+- **subscriptions**
+
+  Number of subscriptions
+
+- **subscriptions per {source ID &| event type}**
+
+  Number of subscriptions per {source ID &| event type}
+
+- **event activations**
+
+  Number of event activations
+
+- **consumers**
+
+  Number of consumers
+  
+- **errors per {app &| tag &| ...}**
```

Comment:
What kind of errors?

Reply:
Golang errors like for example Internal Error during JSON marshaling, maybe this should be labeled with actor/app and any queriable other tags.
Again this should be the responsibility of Error reporting tools, but just to give some insights to the operation team that there is something wrong happens.